spec/swagger.yaml

```diff
@@ -1851,7 +1873,7 @@ paths:
           format: uuid
       responses:
         200:
-          description: OK
+          description: Successful response
```

Comment:
Response description for 200 should be `OK` to be consistent with other responses.

Reply:
UPDATED